src/test/java/ca/corefacility/bioinformatics/irida/ria/unit/web/services/UISampleServiceTest.java

```diff
@@ -0,0 +1,95 @@
+package ca.corefacility.bioinformatics.irida.ria.unit.web.services;
+
+import java.util.Locale;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.context.MessageSource;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContext;
+
+import ca.corefacility.bioinformatics.irida.model.project.Project;
+import ca.corefacility.bioinformatics.irida.model.sample.Sample;
+import ca.corefacility.bioinformatics.irida.model.user.Role;
+import ca.corefacility.bioinformatics.irida.model.user.User;
+import ca.corefacility.bioinformatics.irida.ria.web.samples.dto.SampleDetails;
+import ca.corefacility.bioinformatics.irida.ria.web.samples.dto.ShareSamplesRequest;
+import ca.corefacility.bioinformatics.irida.ria.web.services.UICartService;
+import ca.corefacility.bioinformatics.irida.ria.web.services.UISampleService;
+import ca.corefacility.bioinformatics.irida.security.permissions.sample.UpdateSamplePermission;
+import ca.corefacility.bioinformatics.irida.service.GenomeAssemblyService;
+import ca.corefacility.bioinformatics.irida.service.ProjectService;
+import ca.corefacility.bioinformatics.irida.service.SequencingObjectService;
+import ca.corefacility.bioinformatics.irida.service.sample.SampleService;
+
+import com.google.common.collect.ImmutableList;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class UISampleServiceTest {
+	private UISampleService service;
+	private ProjectService projectService;
+	private final User USER_1 = new User("test", "test@nowhere.com", "PW1@3456", "Test", "Tester", "1234567890");
+	private final Sample SAMPLE_1 = new Sample("SAMPLE_01");
+
+	private final Long SAMPLE_ID = 313L;
+	private final String SAMPLE_ORGANISM = "Salmonella";
+	private final String SAMPLE_DESCRIPTION ="This is a project about interesting stuff";
+
+	@Before
+	public void setUp() {
+		SampleService sampleService = mock(SampleService.class);
+		 projectService = mock(ProjectService.class);
```

Comment:
Remove space in front of `projectService`

Reply:
Removed in 0e6e8fb